test-framework/junit5-component/src/main/java/io/quarkus/test/component/QuarkusComponentTestConfiguration.java

```diff
@@ -75,6 +109,16 @@ QuarkusComponentTestConfiguration update(Class<?> testClass) {
                     }
                 }
             }
+            Class<? extends Converter<?>>[] converters = testAnnotation.configConverters();
+            if (converters.length > 0) {
+                for (Class<? extends Converter<?>> converterClass : converters) {
+                    try {
+                        configConverters.add(converterClass.getDeclaredConstructor().newInstance());
+                    } catch (Exception e) {
```

Comment:
```suggestion
                    } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
```

Nitpicking but I'd catch the specific exceptions here :)

Reply:
And why?

Hm, but we should log the exception stack...

Reply:
Just a weird habit of mine I suppose, you never know what else is present in the c-tor logic that might throw an exception.
I agree with the stack